oar/README.md

```diff
@@ -0,0 +1,58 @@
+# OAR (Openshift Automatic Release)
+## OAR Commandline Tool (CLI)
+- Install
+```
+clone git@github.com:openshift/release-tests.git
```

Comment:
better to git clone xxx, so people can copy it directly

Reply:
done